From 41428a1b3351d9b0e9c1f07421211d934ef53421 Mon Sep 17 00:00:00 2001
From: Eric Naim <dnaim@cachyos.org>
Date: Sat, 10 Jan 2026 11:47:43 +0700
Subject: [PATCH 51/68] hid-msi-claw: Remove the resume callback

This causes devices using this driver to instantly freeze the instant
the device wakes up

Link: https://github.com/CachyOS/CachyOS-Handheld/issues/91
Signed-off-by: Eric Naim <dnaim@cachyos.org>
---
 drivers/hid/hid-msi-claw.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/hid/hid-msi-claw.c b/drivers/hid/hid-msi-claw.c
index 03cf1d449..3d95e31c1 100644
--- a/drivers/hid/hid-msi-claw.c
+++ b/drivers/hid/hid-msi-claw.c
@@ -834,9 +834,6 @@ static struct hid_driver msi_claw_driver = {
 	.raw_event		= msi_claw_raw_event,
 	.probe			= msi_claw_probe,
 	.remove			= msi_claw_remove,
-#ifdef CONFIG_PM
-	.resume			= msi_claw_resume,
-#endif
 };
 module_hid_driver(msi_claw_driver);
 
-- 
2.52.0

